tests/addUserFuncPart2.spec.ts

```diff
@@ -1,62 +1,89 @@
-import { test, expect } from "@playwright/test";
+import { test, expect, APIResponse } from "@playwright/test";
 import { generateRandomUserName } from "../helpers/generateRandomUserName";
 import { URLS } from "../config/urlProvider";
 import { PageFactory } from "../pageFactory/pageFactory";
 import { AddUserPage } from "../pages/addUserPage";
 import { AddUserSteps } from "../steps/addUserSteps";
-
-const invalidYearOfBirth: string[] = [
-  "1899",
-  "1898",
-  (new Date().getFullYear() - 17).toString(),
-  (new Date().getFullYear() - 16).toString(),
+import { UserDto } from "../dto/userDto";
+import { UserApiClient } from "../api/userApiClient";
+import { UserDtoResponse } from "../dto/userDtoResponse ";
+import { containsUser } from "../helpers/containsUser";
+
+const usersWithInvalidYearOfBirth: UserDto[] = [
+  new UserDto(generateRandomUserName(4), "1899"),
```

Comment:
you can use here the same value as your main goal - to create users with wrong year of birth, but name is'not necessary generate with different amount of chars: generateRandomUserName(4)